terraform/modules/aws/route53/main.tf

```diff
@@ -0,0 +1,32 @@
+
+# provider "aws" {
+#   region = "eu-central-1"
+# }
+
+resource "aws_route53_zone" "omega" {
+  name = var.zone_name
+}
+
+resource "aws_route53_record" "record" {
+
+  zone_id = aws_route53_zone.omega.zone_id
+  name    = var.record_name
+  type    = var.record_type
+  #ttl     = var.record_ttl
+  #records = var.record_records
+
+  #we don’t have alb, I specified entries in the alias of the current A record, then we will change it to substitute the values of alb.
+   alias {
+     name = "dualstack.alb-omega-2144136904.eu-west-1.elb.amazonaws.com."
+     zone_id = "eu-west-1"
+     evaluate_target_health = true  
+     }
+ }
+
+resource "aws_route53_record" "cname_record" {
```

Comment:
can we make change this to create multiple CNAME records?

Reply:
I think we can try to change it

Reply:
@ggramal  made it possible to create multiple CNAMEs. I checked - everything works.

Reply:
```
$ terraform plan
╷
│ Error: Missing newline after argument
│ 
│   on dns_viriables.tf line 7, in variable "records":
│    7:   description = "A string of DNS records, example: ["192.158.1.1"]"
│ 
│ An argument definition must end with a newline.
╵
```
:thinking:

Reply:
oh sry my bad its wrong team

Reply:
```
Initializing the backend...
Initializing modules...
- route53 in ../../../modules/aws/route53
- vpcs in 
╷
│ Error: Unreadable module directory
│ 
│ Unable to evaluate directory symlink: lstat ../../../modules/aws/vpc: no such file or directory
╵

╷
│ Error: Unreadable module directory
│ 
│ The directory  could not be read for module "vpcs" at main.tf:36.
╵
```

Reply:
maybe you need to merge master :thinking:

Reply:
Yes, you need to merge this branch into master. I checked it locally - 2 modules completed the terraform plan.
What should we do? We cannot merge until we receive approval. We can push the vpc module into the current brunch so that you can make a terraform plan.

Reply:
so yeah vpc module is in master branch so its in production state. What you need to do is to merge master into this branch

Reply:
@ggramal Corrected, check.